tests/microos/toolbox.pm

```diff
@@ -145,6 +145,8 @@ sub clean_container_host {
 }
 
 sub post_fail_hook {
+    my ($self) = @_;
```

Comment:
FWICT the post_fail_hook is actually useless so could be removed

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18200/commits/5e6749d336c139c75f90551db388ff1f89e85b86